scripts/quality/run_codex_exec.py

```
@@ -124,9 +124,9 @@ def _run_codex_exec(
     # Safe-by-construction: a fixed literal executable name, explicit argv,
     # shell=False, validated tokens as plain arguments, and prompt content
     # flowing only through stdin.
+    # nosemgrep: python.lang.security.audit.dangerous-subprocess-use-audit.dangerous-subprocess-use-audit
     return subprocess.run(  # nosec B603
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)